packages/flutter/test/widgets/navigator_test.dart

```diff
@@ -448,6 +473,161 @@ void main() {
     expect(find.text('B'), findsOneWidget);
   });
 
+  testWidgets('replaceNamed sets secondaryAnimation after transition with history change during transition', (WidgetTester tester) async {
```

Comment:
This test doesn't do anything with replaceNamed?

Reply:
Apologies - that was poor naming of the tests. Changed to pushReplacement.